code/lib/preview-api/src/modules/store/csf/portable-stories.ts

```diff
@@ -104,8 +108,12 @@ export function composeStory<TRenderer extends Renderer = Renderer, TArgs extend
     args: { ...story.initialArgs },
     viewMode: 'story',
     loaded: {},
-    abortSignal: null as unknown as AbortSignal,
+    abortSignal: abortController.signal,
     canvasElement: null,
+    mount: async () => {
+      throw new Error('mount not provided.');
```

Comment:
Can you please make the error more descriptive? Something that tells people it's a Storybook error, and how to fix it.